src/Microsoft.DocAsCode.Build.RestApi/BuildRestApiDocument.cs

```diff
@@ -4,22 +4,23 @@
 namespace Microsoft.DocAsCode.Build.RestApi
 {
     using System;
-    using System.Collections.Generic;
     using System.Composition;
     using System.Collections.Immutable;
+    using System.Collections.Generic;
     using System.IO;
     using System.Linq;
 
     using Microsoft.DocAsCode.Build.Common;
     using Microsoft.DocAsCode.DataContracts.RestApi;
     using Microsoft.DocAsCode.Plugins;
     using Microsoft.DocAsCode.Utility;
-    using Microsoft.DocAsCode.Utility.EntityMergers;
+
+    using Newtonsoft.Json.Linq;
 
     [Export(nameof(RestApiDocumentProcessor), typeof(IDocumentBuildStep))]
     public class BuildRestApiDocument : BaseDocumentBuildStep
     {
-        private readonly ReflectionEntityMerger Merger = new ReflectionEntityMerger();
+        private static readonly HashSet<string> MarkupKeys = new HashSet<string>(StringComparer.OrdinalIgnoreCase) { "description" };

```

Comment:
If ignore case, success to pass in [http://bigstickcarpet.com/swagger-parser/www/index.html](http://bigstickcarpet.com/swagger-parser/www/index.html) while failed in swagger io, better change to case sensitive.

---
In reply to: [86910967](https://github.com/dotnet/docfx/pull/864#discussion_r86910967) [](ancestors = 86910967)